R/path.R

```diff
@@ -224,36 +213,52 @@ path_rel <- function(path, start = ".") {
   path_tidy(path)
 }
 
-#' Paths starting from useful directories
+#' Finding Users Home Directory
+#'
+#' * `path_expand()` performs tilde expansion on a path, replacing instances of
+#' `~` or `~user` with the user's home directory.
+#' * `path_home()` constructs a path within the expanded users home directory
```

Comment:
The utility of calling `path_home()` w/ no arguments to see what fs regards as home directory seems worth pointing out in one or more places.